### DISCUSSION TITLE
Effective range of SOAP tuned by cutoff

### DISCUSSION THREAD

**Discussion by amessl** [SINGROUP/dscribe#149] [Q&A]
Dear DScribe community,
I've been using this library very frequently for my research, primarily for generating SOAP representations. I have observed that for the prediction of the property of my interest, I usually obtain a top combination of SOAP parameters (through cross validation) that uses a cutoff radius of "only" 2.0 A. This is a bit unexpected, because in many of my compounds the average bond length of my central atom to the first coordination sphere is larger than 2.0 A, meaning that these atoms are not accounted for in the resulting representation (?).

From personal correspondence with one of the authors who originally proposed SOAP, I learned that atoms in a range of 2.0 A + sigma(central_atom) from the central atom are included, where sigma is the width of the atom centered gaussians.
So my actual question is: Is the statement above also true for the DScribe implmentation of SOAP ? Does that mean that the "effective" range from some central atom if rcut=2.0 A is actually 3.0 A (because sigma=1.0 is the default in DScribe) ?

I would highly appreciate any clarification.
Kind regards

**Comment by lauri-codes** [comment]
Hi @amessl!

Indeed, any atoms within at least r_cut+sigma contribute to the SOAP output also in our implementation. The effective range might actually be even bigger, I can check a bit later.

**Comment by lauri-codes** [comment]
@amessl: I checked our exact implementation. The effective radius - which determines which atoms to include in the SOAP creation - is set so that any Gaussian decays to 1/1000th of its value at `r_cut`. So to be more precise, the effective cutoff is given by ([see here](https://github.com/SINGROUP/dscribe/blob/893722c28666aa58dadaf083a197a51a71694050/dscribe/descriptors/soap.py#L347)):

```
r_cut + sigma * sqrt(-2 * log(threshold))
```

So if you use `sigma = 1`, `r_cut = 2`, your effective cutoff will be ~5 Å. It is necessary to include atoms outside the cutoff in order to make the SOAP output behave continuously with respect to atom distance from the SOAP center.

I would also highly encourage you to include sigma to your hyperparameter cross-validation: it may have a big impact.
